src/styles/App.scss

```diff
@@ -0,0 +1,237 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&display=swap");
+
+@font-face {
```

Comment:
## Optional: Design(Strongly recommend)
- [ ] Hi, In reading the requirements, I noticed that you left the design behind, tho your App is working fine.
**It says**: You should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Typography: font face, size, and weight.
Layout: composition and space between elements.
- **See Your App**
![Capture](https://user-images.githubusercontent.com/63972670/233206121-6e105b6a-38e3-4657-ba5b-5b44c2d758ac.PNG)


- **See the original design**

![Capture](https://user-images.githubusercontent.com/63972670/233206622-055ab0c1-602a-4267-a093-c5888524b80d.PNG)

- [ ] Above all, I would advise you to work on the design, please follow the Design requirements because this is a capstone. This react capstone can lead you to your dream Job.